software-engineering-policies/SourceControl/RepositorySetupPolicy.md

```diff
@@ -0,0 +1,69 @@
+# Repository Setup Policy
+
+To ensure consistency in how our repositories are setup for access and automation, the following policy has been created. This focuses on the files within the codebase, the settings within the repo and automation we expect to run. 
+
+## Codebase setup
+
+So that we can ensure that anybody interacting with the codebase knows how they can perform certain actions, the following files should be added.
+
+### CODEOWNERS
+
+The CODEOWNERS file uses pattern matching to figure out who should be reviewing what when it comes to a pull request. For a CODEOWNERS file to be valid, the users/teams must have write permission to the repository. More information on how this can be setup and structured can be found in the [github documentation](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)
+
+### LICENSE
+
+This describes the license in which the codebase is under. 
+
+### CONTRIBUTING.md
+
+The contributing file explains to anyone wishing to interact with the codebase how they would go about doing so. This would include raising issues, how to go about creating pull requests and any guidance to what should be included when making changes. Examples of these files can be found [here](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors#examples-of-contribution-guidelines)
```

Comment:
should we link the current contributing guide instead: https://github.com/UKHO/docs/blob/main/using-github/creating-a-contributing-file.md

Reply:
Updated! Thank you for this - if we have already worked on this we should deffo use it